Language/Haskell/GhcMod/Debug.hs

```diff
@@ -138,5 +139,5 @@ mapDoc kd ad m = vcat $
 ----------------------------------------------------------------
 
 -- | Obtaining root information.
-rootInfo :: (IOish m, GmOut m) => m String
-rootInfo = (++"\n") . cradleRootDir <$> findCradle
+rootInfo :: forall m. (IOish m, GmOut m) => m String
+rootInfo = (++"\n") . cradleRootDir <$> fst `liftM` (runJournalT findCradle :: m (Cradle, GhcModLog))
```

Comment:
Can you not just add a GmLog constraint here? I'm not sure the context this is being run in allows it nevermind that if it doesn't.